README.md

```diff
@@ -33,8 +33,6 @@ The attributes listed below are properly formatted as JSON in [*example.json*](h
 
 >**\_start** (string):  Used only on an article, defines the starting block for the branching scenario. Leave blank to use the first block.
 
->**\_containerId** (string):  To add a block to an alternative branching set, add the branching id here. Useful to pull in a block that lives outside of a branching article into that article's branching scenario. Leave blank to use the id of the current parent article.
```

Comment:
The use case for this property is not known or easy to derive. Removal is probably a good idea.